### DISCUSSION TITLE
How to remove/uninstall?

### DISCUSSION THREAD

**Discussion by gitpulled** [d8ahazard/sd_dreambooth_extension#436] [Q&A]
There isn't a remove/uninstall button in the Auto1111 web ui. What is safe to just delete from the stable diffusion folder? 

In regards to using the WebUI to install dreambooth.

**Comment by d8ahazard** [comment]
Yep

**Reply by gitpulled** [reply]
My fault, I thought I was on a specific forum, I think I posted this in a general sense. Updated the OP.
